src/motion/actuators/GenericActuator.hx

```diff
@@ -216,6 +229,31 @@ class GenericActuator<T> implements IGenericActuator {
 	}
 	
 	
+	/**
+	 * Defines a function which will be called when the tween starts
+	 * @param	handler		The function you would like to be called
+	 * @param	parameters		Parameters you would like to pass to the handler function when it is called
+	 * @return		The current actuator instance
+	 */
+	public function onStart (handler:Dynamic, parameters:Array <Dynamic> = null):GenericActuator<T> {
+		
+		_onStart = handler;
+		
+		if (parameters == null) {
+			
+			_onStartParams = [];
+			
+		} else {
+			
+			_onStartParams = parameters;
+			
+		}
+		
+		return this;
+		
+	}
+	
+	
```

Comment:
Isn't that the wrong position when sorted alphapetical?

Reply:
Oh yes that could be. I intentionally organized them in start, update, complete order but it makes sense to organize them in alphabetical order. I'll check if that's the order how the methods are organized.

Reply:
Change the code please have a second look